Formula/git-extras.rb

```diff
@@ -18,13 +18,21 @@ class GitExtras < Formula
     sha256 "59e7af5b051fcab03f394ebb8627d5f41f9b9c3be543b59329c9356e1565eee8" => :sierra
     sha256 "59e7af5b051fcab03f394ebb8627d5f41f9b9c3be543b59329c9356e1565eee8" => :el_capitan
     sha256 "59e7af5b051fcab03f394ebb8627d5f41f9b9c3be543b59329c9356e1565eee8" => :yosemite
+    rebuild 1
```

Comment:
Don't need this line added.

Reply:
Version is not changing so without this line it doesn't rebuild the bottle and will fetch the existing bottle for this specific version.  In the old days I would have marked this `revision 1` but `audit` told me to use `rebuild 1` instead.

Reply:
:+1: taken care of, thanks for the info!